src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.block {
+  width: 100%;
+  height: 100px;
+}
+
+.red {
```

Comment:
for the future, do not name class `red, blue ...` or something like that.

Reply:
Ok, got you.